go/go110/Dockerfile

```diff
@@ -17,6 +17,12 @@ FROM golang:1.10
 # Install python3 for trampoline.
 RUN apt-get update && apt-get install -y python3 && which python3
 
+# Install protoc
+RUN apt-get update && apt-get install -y unzip wget vim
```

Comment:
We should consider putting all of this into a bootstrap script to avoid having to duplicate.

Reply:
I'll do this in a follow-up down the line as well. Created #77 to track this.